Makefile.Qt

```diff
@@ -81,7 +81,7 @@ S7OPTS ?=
 
 WHICHFAUST = $(shell pwd)/bin/packages/faust2/compiler/faust
 # The include files in audio/faudiostream are copied from Faust 0.9.55.
-FAUST = $(WHICHFAUST) -I $(shell pwd)/bin/packages/faust2/architecture  #`which faust` #/usr/bin/faust #~/site/bin/faust
```

Comment:
Removed this one too. I don't think that would have compiled. This is faust compiler options, not c/c++ compiler options.